docs/web/contribute.md

```diff
@@ -43,88 +86,163 @@ assign the issue to yourself. This way, nobody else will work on it at the same
 issue that someone else is assigned to, please contact the assignee to know if they are still working
 on it.
 
+#### Make a branch
 
-## Working with backends
+**Dependencies:**
 
-Ibis comes with several backends. If you want to work with a specific backend, you will have to install
-the dependencies for the backend with `conda install -n ibis-dev -c conda-forge --file="ci/deps/<backend>.yml"`.
+- required: `git`
 
-If you don't have a database for the backend you want to work on, you can check the configuration of the
-continuos integration, where docker images are used for different backend. This is defined in
-`.github/workflows/main.yml`.
+Once you're inside of a nix-shell, the first thing you want to do is make a branch. Let's call it `useful-bugfix`.
```

Comment:
since this is not specific to nix, can you be more generic here

Reply:
Can you be more descriptive in what you're asking for?